04_environment/index.tex

```diff
@@ -120,34 +108,31 @@ \subsubsection{Pareto Optimal Strategy}
 
 \subsubsection{Social Welfare Maximisation}
 
-It is possible for these islands to find the social welfare maximum in this foraging function with enough time. The islands could identify the exact amount of resources needed, for both fishing and deer hunting, to achieve the best return according to the maximum number of animals. Thus by cooperating who goes where and spends how much they are able to maximize their returns without spending more resources than necessary.
-
-\subsection{Extended Work}
-
-For the future implementations, our additions focus on further exploring the dynamics of our agents and allowing for a more complex decision-making interaction between clients and the environment. Some of these design ideas were later implemented in self-organising multi-agent system and others are expressed for the sake of completeness and future work recommendations.
-
-Some changes have been made to make foraging more volatile and unpredictable. These will add to the complexity of the system, allowing the teams to further evaluate the agents’ performance on observing and using learned knowledge to take action.
+It is possible for these islands to find the social welfare maximum in this foraging function with enough time. The islands could identify the exact amount of resources needed, for both fishing and deer hunting, to achieve the best return according to the maximum number of animals. Thus, by cooperating who goes where and spends how much, they are able to maximize their returns without spending more resources than necessary.
```

Comment:
you again use 'welfare'.. 
I think you should define this :) if you keep referring to it

Reply:
`\begin{definition} \end{definition}` would like nice